3.9/arangosearch-performance.md

```diff
@@ -339,3 +339,28 @@ used. You can create custom Analyzers without this feature to disable the
 normalization and improve the performance. Make sure that the result ranking
 still matches your expectations without normalization. It is recommended to
 use normalization for a good scoring behavior.
+
+## Primary key caching
+
+<small>Introduced in: v3.9.6</small>
+
+You can set the `primaryKeyCache` View property to `true` to always cache the
+the primary key columns in memory. This can improve the performance of queries
+that return many documents:
```

Comment:
It would be helpful to better explain the use cases for this option, when it can help in particular, with a small example.

Reply:
The same story as with `"cache":true` for fields. But this time it is about document ids (LocalDocumentId) to get the actual doc for SEARCH.  The only difference that this field used only when producing results. E,g, if there is SORT + LIMIT than less documents will be actually produced than scanned. So this option has less effect on the performance in case of query with "LIMIT 10"

Reply:
I see. Do you think this description is okay?
> This can improve the performance of queries that return many documents, making it faster to map document IDs in
the index to actual documents

Reply:
Yes